setup.py

```diff
@@ -19,7 +19,8 @@
 DEBUG = check_env_flag('DEBUG')
 WITH_DISTRIBUTED = not check_env_flag('NO_DISTRIBUTED')
 WITH_DISTRIBUTED_MW = WITH_DISTRIBUTED and check_env_flag('WITH_DISTRIBUTED_MW')
-WITH_NCCL = WITH_CUDA and platform.system() != 'Darwin'
+# WITH_NCCL = WITH_CUDA and platform.system() != 'Darwin'
+WITH_NCCL = False
```

Comment:
Could we make it conditional on the compilation, so that we can still leverage NCCL on NVidia?